static/js/containers/FinancialAidCalculator.js

```diff
@@ -56,7 +55,7 @@ const currencySelect = (update, current) => (
 
 const salaryUpdate = R.curry((update, current, e) => {
   let newEdit = R.clone(current);
-  newEdit.income = sanitizeNumberString(20, e.target.value);
+  newEdit.income = e.target.value
```

Comment:
why not use the `filterPositiveInt` function we define in `util/util.js`? then we can keep the in-line behavior, preventing the user from typing a decimal in the first place.

you could do something like

```js
const sanitizeIncome = R.compose(filterPositiveInt, sanitizeNumberString(20));

newEdit.income = sanitizeIncome(e.target.value);
```